cli/cmd/sync.go

```diff
@@ -98,12 +98,18 @@ func sync(cmd *cobra.Command, args []string) error {
 
 	for _, cl := range sourcesClients {
 		maxVersion, err := cl.MaxVersion(ctx)
+		destinationClientsForSource := make([]*manageddestination.Client, 0)
```

Comment:
```suggestion
		var destinationClientsForSource []*manageddestination.Client
```
For the sake of simplicity?